examples/main.rs

```diff
@@ -13,10 +13,12 @@ enum Tile {
 
 impl From<Tile> for String {
     fn from(val: Tile) -> Self {
+        // See ANSI escape codes for 8-bit colors:
+        // https://en.wikipedia.org/wiki/ANSI_escape_sequence#8-bit
         match val {
-            Tile::Dirt => "\x1b[1;30m██\x1b[0m".to_string(),
-            Tile::Grass => "\x1b[1;32m██\x1b[0m".to_string(),
-            Tile::Water => "\x1b[1;34m██\x1b[0m".to_string(),
+            Tile::Water => "\x1b[1;34m██".to_string(),
+            Tile::Dirt => "\x1b[38;2;200;150;100m██".to_string(),
```

Comment:
This is a 24-bit sequence and not supported by many terminals. Use one of the 8-bit ones.